### DISCUSSION TITLE
Is this project still mantained?

### DISCUSSION THREAD

**Discussion by alexandremjacques** [tabler/tabler#1876] [General]
Looking at the last release and latest commits, it appears that this project is halted. Any plans on keeping it evolving?

If so, are there any plans for a PRO version with support?

TIA

**Comment by ollym** [comment]
We would be prepared to sponsor Tabler if it would help keep it maintained

**Comment by hany** [comment]
Also came here to ask this. Wondering if this project is still active! Would be willing to sponsor if work will continue.

**Comment by abalter** [comment]
I would as well.

**Comment by abalter** [comment]
I think a few medium articles showing it off would help.

And also some examples other than the 'ol ubiquitous dashboard.

**Comment by lllllllillllllillll** [comment]
@alexandremjacques @ollym  @hany  @abalter 

It is.
Check the commit history: https://github.com/tabler/tabler/commits/dev/ or Paweł's recent contributions: https://github.com/codecalm.
They probably have work saved locally or to a private repo as well, which won't appear until it's merged into the main branch.

**Comment by dylan-tao** [comment]
> Looking at the last release and latest commits, it appears that this project is halted. Any plans on keeping it evolving?
> 
> If so, are there any plans for a PRO version with support?
> 
> TIA
@alexandremjacques @hany @ollym @abalter @lllllllillllllillll  codecalm's author goal is to have 50 monthly sponsors，please mark my answer，thanks！
Sponsored Address： https://github.com/sponsors/codecalm

**Comment by plaimbock** [comment]
I emailed Paweł if he needed help and he invited me to collaborate on the tabler repository. Per Paweł's request, please let me know if you are also interested in helping out so I can send him a list with gh accounts to consider.

**Comment by janstadt** [comment]
Curious about the premium paid features. @plaimbock @codecalm do you know if the paid plan comes with premium support? If we were to pay, would that help in response times for questions/issues? I am proposing this lib as a refresh for my companies internal tooling but i dont want to recommend a lib that is not well supported.

**Comment by codecalm** [comment]
@janstadt I am working hard to make Tabler profitable. lately, after releasing new version of Tabler Emails and Tabler Illustrations, I'm starting to bounce back from the bottom and I'm starting to see a light in the tunnel. 

I am open to any form of cooperation that will help develop the project. First of all, people who would help me to complete the documentation, and to release a stable version 1.0.

**Reply by georgedobre** [reply]
> @janstadt I am working hard to make Tabler profitable. lately, after releasing new version of Tabler Emails and Tabler Illustrations, I'm starting to bounce back from the bottom and I'm starting to see a light in the tunnel.
> 
> I am open to any form of cooperation that will help develop the project. First of all, people who would help me to complete the documentation, and to release a stable version 1.0.

i've been keeping an eye on this project, could we get an insight of the actual things that need doing? there are plenty of people willing to help, myself included

**Reply by janstadt** [reply]
Can i make a suggestion to create a project: https://github.com/tabler/tabler/projects?query=is%3Aopen and set up something similar to this: https://github.com/orgs/music-assistant/projects/2

**Reply by ollym** [reply]
@codecalm I just signed our company up for a $250/month gold sponsorship, may it help make that light at the end of the tunnel a little brighter. The project is so close to 1.0, I hope you find the time for a final push to get it over the finish line. Once it's 1.0 we'd like to use it in production and that would be enough for me to keep sponsoring long term and perhaps even increase to keep it maintained long term.
